### PR TITLE
[2.7] bpo-29521: Fix two minor documentation build warnings (GH-41)

### PR DESCRIPTION
**Pull request by Mariatta**
(cherry picked from commit 3d707be950b387552585451071928e7b39cdfa53)

### CODE REVIEW

File: Doc/faq/windows.rst [hunk 1]
```diff
@@ -294,9 +294,10 @@ this respect, and is easily configured to use spaces: Take :menuselection:`Tools
 --> Options --> Tabs`, and for file type "Default" set "Tab size" and "Indent
 size" to 4, and select the "Insert spaces" radio button.
 
-If you suspect mixed tabs and spaces are causing problems in leading whitespace,
-run Python with the :option:`-t` switch or run ``Tools/Scripts/tabnanny.py`` to
-check a directory tree in batch mode.
+Python raises :exc:`IndentationError` or :exc:`TabError` if mixed tabs 
+and spaces are causing problems in leading whitespace.
```
Comment (vadmium) [inline Doc/faq/windows.rst]: In Python 2, you only get exceptions if you enable the "-tt" option. There shouldn’t be any Sphinx warning for Python 2.

**Comment by Mariatta** [comment]
@vadmium I cleaned up the trailing whitespace that travis build was complaining about.
Is this ok to merge now? Thanks :)

**Comment by vadmium** [comment]
Sorry I was actually talking about the original warning that I reported <https://bugs.python.org/issue29387>:

faq/windows.rst:303: WARNING: unknown option: -t

This pull request is for the 2.7 branch. As far as I remember, the above warning should not be present on the 2.7 branch, because the “-t” option is useful and is documented at <https://docs.python.org/2.7/using/cmdline.html#cmdoption-t>. Plus the new text is wrong: by default, Python 2 will accept code with mixed tabs and spaces in some cases.

I suggest just fixing the second half of the sentence:

```rst
If you suspect mixed tabs and spaces are causing problems in leading whitespace,
run Python with the :option:`-t` switch or run the :mod:`tabnanny` module to
check a directory tree in batch mode.
```

(I would have done this myself, but I haven’t caught up with all the Git Hub changes.)

**Comment by Mariatta** [comment]
@vadmium I updated the wordings as suggested. Please let me know if this works :)

**Comment by vadmium** [comment]
The tabnanny change looks fine to me. I have no idea if the Sphinx config is appropriate for 2.7 though.

**Pull request merged**